lib/render.js

```diff
@@ -47,16 +49,38 @@ module.exports = function(options, emitter) {
 
     emitter.emit('warn', chalk.green('Rendering Complete, saving .css file...'));
 
-    fs.writeFile(options.dest, result.css, function(err) {
-      if (err) {
-        return emitter.emit('error', chalk.red(err));
-      }
+    // Check if output directory exists
+    var dest_dir = path.dirname(path.normalize(options.dest));
+    var write_out = function() {
```

Comment:
Functions are written using camelCase like `function writeOut()` and not using `var`.